changelogs/unreleased/gh-6452-mvcc-crash-in-prepare.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/core
+
+* Fix a crash in mvcc connected with secondary index conflict (gh-6452)
```

Comment:
Nit: please add dots to the end of sentences in all your changelogs — not sure changelog reviewers are happy with this.

Reply:
Also, please use the past sense for all changelogs, see https://www.tarantool.io/en/doc/latest/contributing/release_notes/